app/controllers/admin/base_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Admin
+  class BaseController < ApplicationController
+    layout 'admin'
+    before_action :authenticate_user!
+    before_action :admin_user!
+
+    private
+
+    def admin_user!
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not 0) spaces for rails indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)

<!-- Comment by [SideCI](https://sideci.com) -->